handbook/stylesheets/extra.css

```diff
@@ -17,16 +17,137 @@
   --md-footer-fg-color--lighter:     #03b5869f;
 }
 
+/* Navigation bars (left and right) */
+
+.md-nav {
+  color: #333;
+  font-size: 16px;
+}
+
+.md-nav label {
+  font-weight: 700;
+  text-transform: uppercase;
+  font-size: 18px;
+  font-family: "Roboto Condensed", sans-serif;
```

Comment:
We've typically tried to alphabetize properties like this where we can. Applies throughout.